repos/spack_repo/builtin/packages/py_courlan/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack_repo.builtin.build_systems.python import PythonPackage
+
+from spack.package import *
+
+
+class PyCourlan(PythonPackage):
+    """Clean, filter and sample URLs to optimize data collection:
+    deduplication, spam, content and language filters."""
+
+    homepage = "https://github.com/adbar/courlan"
+    pypi = "courlan/courlan-1.3.2.tar.gz"
+
+    version("1.3.2", sha256="0b66f4db3a9c39a6e22dd247c72cfaa57d68ea660e94bb2c84ec7db8712af190")
+
+    depends_on("python@3.8:", type=("build", "run"))
+    depends_on("py-setuptools", type="build")
```

Comment:
Per https://github.com/adbar/courlan/blob/v1.3.2/pyproject.toml:


```suggestion
    depends_on("py-setuptools@61:", type="build")
```